gammapy/data/tests/test_data_store.py

```diff
@@ -173,17 +174,30 @@ def test_hdu_table(self):
 
     def test_observation(self, monkeypatch):
         """Check that one observation can be accessed OK"""
+        # Note: IRF access requires the CALDB env var
+        caldb_path = Path(os.environ["GAMMAPY_DATA"]) / Path("cta-1dc/caldb")
```

Comment:
This had to be moved up, because the `DataStore.obs` method already loads the IRFs.